src/amchart/CommonSerial.js

```diff
@@ -252,6 +252,9 @@
             dataObjArr.sort(function (a, b) {
                 return a[sortField] - b[sortField];
             });
+            this.data().sort(function (a, b) {
+                return a[0] - b[0];
+            });
```

Comment:
I can't see how this will work, when your dates look like "03 Jan 2015", "01 July 2016" etc...